### PR TITLE
implement settings page

### PR DESCRIPTION
**Pull request by ericcf**

### CODE REVIEW

File: src/css/application.css [hunk 1]
```diff
@@ -16,6 +16,22 @@ body {
   padding-top: 68px;
 }
 
+body.font-size-18 {
```
Comment (michaelwehrley) [inline src/css/application.css]: Are you using bootstrap?  What happens to tags such as h5 which may be smaller than the selected font size but will not be overridden?

Comment (ericcf) [inline src/css/application.css]: That's a good question. This feature is a legacy one being ported from the Android app. It was never very well thought out. @cbrenner04 may be able to speak to whether it's even still a requirement of the app.

Comment (cbrenner04) [inline src/css/application.css]: Are you asking if the settings page is still a requirement? If so, as far as I know, yes. I would say if there isn't a significant difference between the two apps in regards to this then just leave it.

**Comment by michaelwehrley** [comment]
:+1:

**Pull request merged**